starknet_py/tests/e2e/client/full_node_test.py

```diff
@@ -400,3 +413,179 @@ async def test_get_syncing_status(full_node_client):
         sync_status = await full_node_client.get_syncing_status()
 
     assert isinstance(sync_status, SyncStatus)
+
+
+# ---------------------------- Trace API tests
+
+
+@pytest.mark.asyncio
+async def test_simulate_transactions_skip_validate(
+    full_node_account, deployed_balance_contract
+):
+    assert isinstance(deployed_balance_contract, Contract)
+    call = Call(
+        to_addr=deployed_balance_contract.address,
+        selector=get_selector_from_name("increase_balance"),
+        calldata=[0x10],
+    )
+    invoke_tx = await full_node_account.sign_invoke_transaction(
+        calls=call, auto_estimate=True
+    )
+
+    simulated_txs = await full_node_account.client.simulate_transactions(
+        transactions=[invoke_tx], skip_validate=True, block_number="latest"
+    )
+
+    assert simulated_txs[0].transaction_trace.validate_invocation is None
```

Comment:
Maybe instead test this with transaction that has no signature like we do in jvm and swift sdks? 
Otherwise it doesn't seem to differ much from other tests that don't have `skip_validate`.